src/nvim/path.c

```diff
@@ -356,11 +356,20 @@ char *concat_fnames(const char *fname1, const char *fname2, bool sep)
  * Add a path separator to a file name, unless it already ends in a path
  * separator.
  */
```

Comment:
Please change this to the new-style comment, like on the function above, and add `/// @pre The buffer p points to is at least MAXPATHL.`